ecpairing/src/lib.rs

```diff
@@ -25,12 +25,6 @@ pub extern "C" fn main() {
     let input = ewasm_api::unsafe_calldata_copy(0, length);
 
     let mut output = [0u8; 32];
-    match ethereum_bn128::bn128_pairing(&input[..], &mut BytesRef::Fixed(&mut output[..])) {
-        Ok(_) => {
-            ewasm_api::finish_data(&output);
-        }
-        Err(_) => {
-            ewasm_api::revert();
-        }
-    }
+    ethereum_bn128::bn128_pairing(&input[..], &mut BytesRef::Fixed(&mut output[..])).unwrap();
```

Comment:
Sorry I actually meant to keep the original structure, just use `panic!()` instead of `revert` so that we can replace that later when ewasm-rust-api has the method added.

Reply:
k, I'm going to finish testing first and I'll update it accordingly.